src/services/date-formatter/date-formatter-service.js

```diff
@@ -1,6 +1,10 @@
-function getLocalDate(timestamp) {
-  const utcDateObj = new Date(timestamp); // remember, JavaScript dates are localized to the user but reflected in UTC time...
-  const OFFSET_MILLIS = new Date().getTimezoneOffset() * 60 * 1000; // get the local offset
+function getLocalDate(timestamp, utcOffset) {
+  // remember, JavaScript dates are localized to the user but reflected in UTC time...
+  // so we use the offset (from Meetup.com) if provided, other use local browser time
+  const utcDateObj = new Date(timestamp);
+  const OFFSET_MILLIS = utcOffset
+    ? utcOffset
+    : new Date().getTimezoneOffset() * 60 * 1000;
 
   return new Date(utcDateObj.getTime() - OFFSET_MILLIS);
```

Comment:
Change to addition `return new Date(utcDateObj.getTime() + OFFSET_MILLIS);`

And remove the `Math.abs`

Meetup provides a signed value such as `-14400000` for the east coast so adding a negative number would result in a subtraction.